scripts/hudson/narayana.sh

```diff
@@ -443,7 +443,7 @@ function init_jboss_home {
 function xts_as_tests {
   echo "#-1. XTS AS Tests"
   cd $WILDFLY_CLONED_REPO
-  ./build.sh -f xts/pom.xml -B $IPV6_OPTS -Dtimeout.factor=300 -Dsurefire.forked.process.timeout=12000 -Dsurefire.extra.args='-Xmx512m' -Djboss.dist="$JBOSS_HOME" -Dversion.org.jboss.narayana=${NARAYANA_CURRENT_VERSION} "$@" test
+  ./build.sh -f xts/pom.xml -B $IPV6_OPTS -Dtimeout.factor=300 -Dsurefire.forked.process.timeout=12000 -Dsurefire.extra.args='-Xmx512m' -Djboss.dist="$JBOSS_HOME" -Dversion.org.jboss.narayana=${NARAYANA_CURRENT_VERSION} $CODE_COVERAGE_ARGS "$@" test
```

Comment:
Why didn't you also add the $CODE_COVERAGE_ARGS to the build lines in the functions lra_tests? Your comment in the PR just says that you skipped doing AS_TESTS and LRA_AS_TESTS.

Reply:
That is actually already in the script: https://github.com/jbosstm/narayana/blob/main/scripts/hudson/narayana.sh#L501C76-L501C95. I am not sure why currently in `main` we can find `LRA_TESTS=0` in the JACOCO PROFILE - possibly at the time when the code coverage args were added into the lra_tests the person maybe didn't realise they needed to change the profile above? Or maybe it was added but was more "work in progress" or maybe even the args were added and there was `LRA_TESTS=1` added too, but at some later point someone changed it to `LRA_TESTS=0`.